site/docs/plugins/chat-members.md

```diff
@@ -0,0 +1,132 @@
+# Chat Members Plugin (`chat-members`)
+
+Automatically store information about users on a chat and retrieve it easilly
+
+## Introduction
+
+In many situations, it is necessary for a bot to have information about all the users of a given chat.
+Currently, though, the Telegram Bot API exposes no method that allows us to retrieve this information.
+
+This plugin comes to the rescue: automatically listening to `chat_member` events and storing the `ChatMember` object.
+It then allows you to use `ctx.chatMembers.getChatMember()` to obtain information about a specific chat member, taking the previously stored information into account.
+
+## Usage
+
+### Storing Chat Members
+
+You can use a valid grammY [storage adapter](https://grammy.dev/plugins/session.html#known-storage-adapters) or an
+instance of any class that implements the [`StorageAdapter`](https://deno.land/x/grammy/mod.ts?s=StorageAdapter) interface.
+
+<CodeGroup>
+  <CodeGroupItem title="TypeScript" active>
+
+```ts
+import { Bot, Context, MemorySessionStorage } from "grammy";
+import type { ChatMember } from "grammy/types";
+import { chatMembers, ChatMembersFlavor } from "@grammyjs/chat-members";
+
+type MyContext = Context & ChatMembersFlavor;
+
+const adapter = new MemorySessionStorage<ChatMember>();
+
+const bot = new Bot<MyContext>("<your bot token>");
+
+bot.use(chatMembers(adapter));
+
+bot.start({
+  allowed_updates: ["chat_member", "message"],
+  onStart: ({ username }) => console.log(`Listening as ${username}`),
+});
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="JavaScript" active>
+
+```js
+import { Bot, Context, MemorySessionStorage } from "grammy";
+import { chatMembers, ChatMembersFlavor } from "@grammyjs/chat-members";
+
+const adapter = new MemorySessionStorage();
+
+const bot = new Bot("<your bot token>");
+
+bot.use(chatMembers(adapter));
+
+bot.start({
+  allowed_updates: ["chat_member", "message"],
+  onStart: ({ username }) => console.log(`Listening as ${username}`),
+});
+```
+
+</CodeGroupItem>
+
+<CodeGroupItem title="Deno" active>
+
+```ts
+import {
+  Bot,
+  Context,
+  MemorySessionStorage,
+} from "https://deno.land/x/grammy/mod.ts";
+import type { ChatMember } from "https://deno.land/x/grammy/types.ts";
+import {
+  chatMembers,
+  ChatMembersFlavor,
+} from "https://deno.land/x/grammy_chat_members/mod.ts";
+
+type MyContext = Context & ChatMembersFlavor;
+
+const adapter = new MemorySessionStorage<ChatMember>();
+
+const bot = new Bot<MyContext>("<your bot token>");
+
+bot.use(chatMembers(adapter));
+
+bot.start({
+  allowed_updates: ["chat_member", "message"],
+  onStart: ({ username }) => console.log(`Listening as ${username}`),
+});
+```
+
+</CodeGroupItem>
+
+</CodeGroup>
+
+### Reading Chat Members
+
+This plugin also adds a new `ctx.chatMembers.getChatMember` function that will check the storage for information about a chat member before querying Telegram for it.
```

Comment:
As mentioned above. Shouldn't be presented as new information.

Alternatively, remove it from above.